lib/permit/permissions/condition.ex

```diff
@@ -122,8 +122,12 @@ defmodule Permit.Permissions.Condition do
   def satisfied?(%Condition{condition: function, condition_type: :function_1}, record, _subject),
     do: !!function.(record)
 
-  def satisfied?(%Condition{condition: _fun, condition_type: :function_2}, _record, subject) when is_nil(subject),
-    do: raise "Unable to use function/2 condition feature without subject. First argument of this function is nonexisting subject"
+  def satisfied?(%Condition{condition: _fun, condition_type: :function_2}, _record, subject)
+      when is_nil(subject),
+      do:
+        raise(
+          "Unable to use function/2 condition feature without subject. First argument of this function is nonexisting subject"
```

Comment:
I'd rephrase it:
"Authorization subject cannot be nil when defining conditions using fun/2."

Maybe it could be good to include a short `inspect(..., limit: ...)` snippet of `fun` and `record` here for easier debugging?